docker/deploy/Dockerfile

```diff
@@ -7,4 +7,4 @@ ADD git-rev /ray/git-rev
 WORKDIR /ray/python
 RUN python setup.py install
 WORKDIR /ray
-RUN echo "tail -f /dev/null" >> scripts/start_ray.sh
```

Comment:
Ok, is this correct?
- The command `tail -f /dev/null` just hangs forever.
- So we assume that someone will start the ``ray/deploy` container by running something like
    ```
    docker run ray-project/deploy /ray/scripts/start_ray.sh --head
    ```
    but the problem is that when `start_ray.sh` finishes, the container exits.
- So by appending a command that hangs forever to the end of `start_ray.sh`, we ensure that the container stays alive.